app.js

```diff
@@ -113,6 +114,7 @@ app.use('/screenshot', screenshotRouter);
 app.use('/recaptcha_solver', recaptchaSolverRouter);
 app.use('/mhtml', mhtmlRouter);
 app.use('/har', harRouter);
+app.use('/form_action', formActionRouter);
```

Comment:
Again, maybe this should be called other way. Look in the client part.

Reply:
Done! :)